src/functionalTest/java/uk/gov/hmcts/reform/controllers/SampleFunctionalTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.labs.louisehuytonspring2.controllers;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
```

Comment:
you shouldn't need this
```suggestion
@SpringBootTest
```